repo/packages/K/kong/1/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "packagingVersion": "3.0",
+  "name": "kong",
+  "version": "0.10.3",
+  "scm": "https://github.com/Mashape/kong",
+  "maintainer": "https://getkong.org/community/",
```

Comment:
Hi @ryadav88 we don't have email support for our open source users - we only offer email support to our Enterprise customers (I'm guessing this is the same as many other commercial open source companies - heck, it is probably the same for Mesosphere!). As such, the URL provided by @shashiranjan84 is the best way for folks to get support. Is that acceptable?